### DISCUSSION TITLE
Get an unique identifier of the current QFIeldCloud user/collaborator

### DISCUSSION THREAD

**Discussion by MonikaShah** [opengisch/QField#3219] [Q&A]
How can I get the ID of the collaborator ?

**Comment by suricactus** [comment]
There are ways, bu we do not guarantee the uniqueness of the collaborator id. Use the collaborator username.

**Comment by MonikaShah** [comment]
Please let  me know how to use that

On Sat, Aug 20, 2022 at 1:52 PM Ivan Ivanov ***@***.***>
wrote:

> There are ways, bu we do not guarantee the uniqueness of the collaborator
> id. Use the collaborator username.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/opengisch/QField/discussions/3219#discussioncomment-3436318>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AA76P5O2A7M57ATQR234AL3V2CIVHANCNFSM57C2HKUA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
With Regards,
Monika Shah

**Reply by suricactus** [reply]
Please read the answer of a question you already asked: https://github.com/opengisch/QField/discussions/3207
